web_src/js/features/repo-issue.js

```diff
@@ -439,16 +439,17 @@ export function initRepoPullRequestReview() {
     $(`#show-outdated-${id}`).removeClass('hide');
   });
 
-  $(document).on('click', 'button.comment-form-reply', function (e) {
+  $(document).on('click', 'button.comment-form-reply', async function (e) {
     e.preventDefault();
```

Comment:
IIFE needed for this `preventDefault` I think but I think the code may as well work with removal of this line it as button click should not have a default action.

Reply:
Also [this rule](https://github.com/github/eslint-plugin-github/blob/main/docs/rules/async-preventdefault.md) should warn on such issues but I guess it can only work with standard `addEventListener`, not jQuery. We really need to get rid of jQuery event listeners :)

Reply:
IIRC, it would submit a form?